server/src/test/java/com/objectcomputing/checkins/services/guild/member/GuildServicesImplTest.java

```diff
@@ -15,7 +15,10 @@
 import org.mockito.Mockito;
```

Comment:
Not a product of this story specifically, but is there any particular reason the class name doesn't match the file name?

Reply:
not sure. fixed though